<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1126</width>
    <height>837</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QCommandLinkButton" name="tm1">
   <property name="geometry">
    <rect>
     <x>2</x>
     <y>180</y>
     <width>409</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Транспонирование матрицы</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>120</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Матрицы</string>
   </property>
  </widget>
  <widget class="QCommandLinkButton" name="tm1_2">
   <property name="geometry">
    <rect>
     <x>2</x>
     <y>220</y>
     <width>409</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Сложение/Вычитание/Умножение матриц</string>
   </property>
  </widget>
  <widget class="QCommandLinkButton" name="tm1_3">
   <property name="geometry">
    <rect>
     <x>2</x>
     <y>260</y>
     <width>409</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Нахождение определителя матрицы</string>
   </property>
  </widget>
  <widget class="QCommandLinkButton" name="tm1_4">
   <property name="geometry">
    <rect>
     <x>2</x>
     <y>300</y>
     <width>409</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Нахождение обратной матрицы</string>
   </property>
  </widget>
  <widget class="QCommandLinkButton" name="tm1_5">
   <property name="geometry">
    <rect>
     <x>2</x>
     <y>340</y>
     <width>409</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Решение линейного уравнения ax = b
 для неизвестного x квадратной матрицы a </string>
   </property>
  </widget>
  <widget class="QCommandLinkButton" name="tm1_6">
   <property name="geometry">
    <rect>
     <x>2</x>
     <y>400</y>
     <width>409</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Решение уравнения ax = b,
где a - ленточная матрица</string>
   </property>
  </widget>
  <widget class="QCommandLinkButton" name="tm1_7">
   <property name="geometry">
    <rect>
     <x>2</x>
     <y>460</y>
     <width>409</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Решение уравнения ax = b,
где a - положительно определённая
 ленточная матрица</string>
   </property>
  </widget>
  <widget class="QCommandLinkButton" name="tm1_8">
   <property name="geometry">
    <rect>
     <x>2</x>
     <y>510</y>
     <width>409</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Решение уравнения ax = b для x,
где a - треугольная матрица</string>
   </property>
  </widget>
  <widget class="QCommandLinkButton" name="tm1_9">
   <property name="geometry">
    <rect>
     <x>2</x>
     <y>570</y>
     <width>409</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Рекурсия Левинсона для Теплицевой матрицы</string>
   </property>
  </widget>
  <widget class="QCommandLinkButton" name="tm1_10">
   <property name="geometry">
    <rect>
     <x>2</x>
     <y>610</y>
     <width>409</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Вычисление уравнения Ax = b
 методом наименьших квадратов</string>
   </property>
  </widget>
  <widget class="QCommandLinkButton" name="tm1_11">
   <property name="geometry">
    <rect>
     <x>2</x>
     <y>660</y>
     <width>409</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Вычисление псевдо-обратной матрицы</string>
   </property>
  </widget>
  <widget class="QCommandLinkButton" name="tm1_12">
   <property name="geometry">
    <rect>
     <x>2</x>
     <y>700</y>
     <width>409</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Нахождение произведения Кронекера</string>
   </property>
  </widget>
  <widget class="QCommandLinkButton" name="tm1_13">
   <property name="geometry">
    <rect>
     <x>2</x>
     <y>740</y>
     <width>409</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Вычисление угла подпространства
между двумя матрицами</string>
   </property>
  </widget>
  <widget class="QCommandLinkButton" name="tm1_14">
   <property name="geometry">
    <rect>
     <x>2</x>
     <y>790</y>
     <width>409</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Проверка двухмерного массива на симметричность</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
